lib/scrabble.rb

```diff
@@ -1,9 +1,33 @@
 class Scrabble
 
-  def score(word)
-    1
+
+
+  def score(word = nil)
```

Comment:
This method is a bit long. I'd love to see you push yourself to break out any nested ifs/blocks/loops (in this case, the `#each` block perhaps) into their own method; this helps clean up your method, work towards SRP & improve encapsulation.